src/actions/schemes.js

```diff
@@ -40,17 +44,30 @@ export function createScheme(scheme: Scheme) {
     );
 }
 
-export function deleteScheme(schemeId: string) {
-    return bindClientFunc(
-        Client4.deleteScheme,
-        SchemeTypes.DELETE_SCHEME_REQUEST,
-        [SchemeTypes.DELETED_SCHEME, SchemeTypes.DELETE_SCHEME_SUCCESS],
-        SchemeTypes.DELETE_SCHEME_FAILURE,
-        schemeId
```

Comment:
`bindClientFunc` wasn't sending `schemeId`?

Reply:
bindClientFunc returns the data returned from the server, and the server doesn't return anything on delete.

Reply:
@jespino Right. The server deletion was tested but not the deletion from redux. Good fix.